pkg/virtual/workspaces/cmd/cmd.go

```diff
@@ -61,13 +61,13 @@ func (o *WorkspacesSubCommandOptions) AddFlags(flags *pflag.FlagSet) {
 	}
 
 	flags.StringVar(&o.KubeconfigFile, "workspaces:kubeconfig", "", ""+
-		"The kubeconfig file of the organizational cluster that provides the workspaces and related RBAC rules.")
+		"The kubeconfig file whose default context points to the root organization workspace.")
```

Comment:
what is the root org workspace? `root` ?

Actually I would expect us to just pass the base config and the code add any `/cluster/foo` suffix necessary.

Reply:
You're right. Just a question of the description not having been updated correctly. The behavior in any case is that we reset the cluster path to empty string to come back to the base config.
Updating the description.

Reply:
Fixed in commit https://github.com/kcp-dev/kcp/pull/613/commits/56ed20b97b626a49852865ad3dbb8ee08284fba7